docs/src/main/asciidoc/extension-metadata.adoc

```diff
@@ -0,0 +1,170 @@
+metadata:
+  short-name: "resteasy-reactive"
+  keywords:
+  - "jaxrs"
+  - "web"
+  - "rest"
+  categories:
+  - "web"
+  - "reactive"
+  status: "stable"
+  guide: "https://quarkus.io/guides/resteasy-reactive"
+  codestart:
+    name: "resteasy-reactive"
+    languages:
+    - "java"
+    - "kotlin"
+    - "scala"
+    artifact: "io.quarkus:quarkus-project-core-extension-codestarts::jar:999-SNAPSHOT"
+  config:
+  - "quarkus.resteasy-reactive."
+  built-with-quarkus-core: "999-SNAPSHOT" <1>
+  capabilities: <2>
+    provides:
+    - "io.quarkus.rest"
+    - "io.quarkus.resteasy.reactive"
+  extension-dependencies: <3>
+  - "io.quarkus:quarkus-resteasy-reactive-common"
+  - "io.quarkus:quarkus-mutiny"
+  - "io.quarkus:quarkus-smallrye-context-propagation"
+  - "io.quarkus:quarkus-vertx"
+  - "io.quarkus:quarkus-arc"
+  - "io.quarkus:quarkus-netty"
+  - "io.quarkus:quarkus-vertx-http"
+  - "io.quarkus:quarkus-core"
+  - "io.quarkus:quarkus-jsonp"
+description: "A JAX-RS implementation utilizing build time processing and Vert.x.\
+  \ This extension is not compatible with the quarkus-resteasy extension, or any of\
+  \ the extensions that depend on it." <4>
+----
+
+<1> Quarkus version the extension was built with
+<2> https://quarkus.io/guides/capabilities[Capabilities] this extension provides
+<3> Direct dependencies on other extensions
+<4> Description that can be displayed to users. In this case, the description was copied from the `pom.xml` of the extension module but it could also be provided in the template file.
+
+[[quarkus-extension-properties]]
+== META-INF/quarkus-extension.properties
+
+Every **runtime** extension artifact **must** include this file. This file is read by the Quarkus bootstrap mechanism and is meant to contain only the information that is relevant for **building or bootstrapping** an applications. This file is typically generated by the corresponding Quarkus Maven or Gradle plugin when the extension project is built and shouldn't be editted manually.
+The following properties may appear in this file:
+
+[cols=3*,options="header"]
+|===
+| Name
+| Presence
+| Description
+
+| `deployment-artifact`
+| Required
+| Points to the corresponding extension deployment artifact in the `groupId:artifactId[:classifier:type]:version` format
+
+| `parent-first-artifacts`
+| Optional
+| Comma-separated list of artifact keys (`groupId:artifactId:classifier:type`) that are to be loaded in a parent first manner. This can be used to work around issues where a given class needs to be loaded by the system ClassLoader. 
+
+| `runner-parent-first-artifacts`
+| Optional
+| Command-separated list of artifact keys that are to be loaded in a parent first manner in addition to those configured with `parent-first-artifacts` when an application is launched from a production binary package. This can be used to work around issues where a given class needs to be loaded by the system ClassLoader. 
```

Comment:
```suggestion
| Comma-separated list of artifact keys that are to be loaded in a parent first manner in addition to those configured with `parent-first-artifacts` when an application is launched from a production binary package. This can be used to work around issues where a given class needs to be loaded by the system ClassLoader. 
```